README.md

```diff
@@ -188,6 +188,8 @@ jobs:
 | `api-url`      | No       | GitHub App URL. You wouldn't need to set this unless you're deploying a custom GitHub app instead of [optic-release-automation](https://github.com/apps/optic-release-automation). <br /> (_Default: `https://optic-release-automation-ocrlhra4va-ue.a.run.app/`_) |
 | `sync-semver-tags`| No    | If you want to keep the major and minor versions git tags synced to the latest appropriate commit <br /> (_Default: `false`_)                                                                  |
 | `notify-linked-issues`| No       | An optional flag to enable an automatic comment on all issues linked to each release so that people following those issues get notified of the code being released. <br /> (_Default: `true`_)                                                                  |
+| `artifact-path`| No       | If this property is populated, the item (folder or file) specified here will be used as the artifact to zip and attach to the release.                                                                  |
+| `artifact-filename`| No       | The artifact file name with the specific extension.                                                                 |
```

Comment:
it's not clear from here what this input does

Reply:
I tried to change the wording in order to be more clear